person_id: person
robots: 
  - id: robot1
    default_loc: [500, 500]
  - id: robot2
    default_loc: [502, 500]
  - id: robot3
    default_loc: [504, 500]
  - id: robot4
    default_loc: [506, 500]
  - id: robot5
    default_loc: [508, 500]
ball_id: ball
experiments:
  - start_x: -10.7
    start_y: 15.4
    start_yaw: -0.135527
    ball_x: 38.15
    ball_y: 9.25
    path:
      - id: 45
        robot: no
      - id: 54
        robot: no
      - id: 61
        robot: yes
        robot_location: [-2.55, 15.8]
        robot_yaw: 0.0129863
      - id: 58
        robot: no
      - id: 46
        robot: yes
        robot_location: [7.2, 15.25]
        robot_yaw: 0.0434509
      - id: 39
        robot: no
      - id: 32
        robot: yes
        robot_location: [8.75, 10.6]
        robot_yaw: -1.5458
      - id: 36
        robot: no
      - id: 37
        robot: no
      - id: 33
        robot: no
      - id: 34
        robot: yes
        robot_location: [23.4, 11.8]
        robot_yaw: 0.26791
      - id: 27
        robot: yes
        robot_location: [24.4, 8]
        robot_yaw: -1.71269
      - id: 20
        robot: no
      - id: 21
        robot: no
      - id: 12
        robot: no
      - id: 13
        robot: no
      - id: 22
        robot: no
    max_duration: 120 #secondsrobot_yaw = self.robots_in_experiment
